File: src/practice/CodingQues/LeetCode/P70ClimbingStairs.java
package practice.CodingQues.LeetCode;

import java.util.Scanner;

public class P70ClimbingStairs {
    public static void main(String[] args) {
        System.out.println("climbStairs(new Scanner(System.in).nextInt()) = " + climbStairs(new Scanner(System.in).nextInt()));
    }

    public static int climbStairs(int n) {
        int x, y;
        x = 1;
        y = 1;
        for (int i = 0; i < n - 1; i++) {
            System.out.println(i);
            int temp = x;
            x = x + y;
            y = temp;
        }
        return x;
    }
}

File: src/practice/CodingQues/LeetCode/P70ClimbingStairs.py
class Solution:
    def climbStairs(self, n: int) -> int:
        x = 1
        y = 1
        for i in range(n - 1):
            temp = x
            x = x + y
            y = temp
        return x


a = int(input("Enter int"))
print(Solution.climbStairs(a))
